### PR TITLE
GitHub workflow: Improve Milestone

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Do not create card when there already exist one to prevent Action error and email with an error log.

Do not add card for closed issue, it's already done.

### CODE REVIEW

File: .github/workflows/MilestoneIssue.yml [hunk 1]
```diff
@@ -16,7 +18,30 @@ jobs:
...
             const TODO_COLUMN = 4971951;
-            console.log("Creating Issue card");
-            // Action will fail if a card already exists for this issue
-            github.projects.createCard({ column_id: TODO_COLUMN, content_id: context.payload.issue.id, content_type: "Issue" });
+            const IN_PROGRESS_COLUMN = 4971952;
+            const REVIEW_IN_PROGRESS_COLUMN = 4971953;
+            const REVIEW_APPROVED_COLUMN = 4971954;
+            const DONE_COLUMN = 4971955;
+            //
+            async function findCard(content_url) {
```
Comment (pavel-mikula-sonarsource) [inline .github/workflows/MilestoneIssue.yml:26]: This method is copy/pasted from here, except different column ordering:
https://github.com/SonarSource/sonar-dotnet/blob/9bf155c48565506572195db880001524699662f4/.github/workflows/RequestReview.yml#L39-L53

**Comment by pavel-mikula-sonarsource** [comment]
I'm going to ignore the failed CFG repox step

File: .github/workflows/MilestoneIssue.yml [hunk 2]
```diff
@@ -16,7 +16,30 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const TODO_COLUMN = 4971951;
-            console.log("Creating Issue card");
-            // Action will fail if a card already exists for this issue
-            github.projects.createCard({ column_id: TODO_COLUMN, content_id: context.payload.issue.id, content_type: "Issue" });
+            const IN_PROGRESS_COLUMN = 4971952;
+            const REVIEW_IN_PROGRESS_COLUMN = 4971953;
+            const REVIEW_APPROVED_COLUMN = 4971954;
```
Comment (andrei-epure-sonarsource) [inline .github/workflows/MilestoneIssue.yml:23]: also declared `const VALIDATE_PEACH_COLUMN = 7838551;`?

you have it in `findCard()` but not here

**Review by andrei-epure-sonarsource — CHANGES_REQUESTED** [review]

File: .github/workflows/MilestoneIssue.yml (hunk 2 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/MilestoneIssue.yml:23]: Good catch 🤦‍♂️

**Review by andrei-epure-sonarsource — APPROVED** [review]
lgtm

**Pull request merged**